### PR TITLE
ArtisteerLimited.Nicepage version 4.0.3.

### PR DESCRIPTION
**Pull request by zephiowl**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you [validated](https://github.com/denelon/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/36245)

### CODE REVIEW

File: manifests/a/ArtisteerLimited/Nicepage/4.0.3.4.0.3/ArtisteerLimited.Nicepage.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 0.4.3.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: ArtisteerLimited.Nicepage
+PackageVersion: 4.0.3.4.0.3
```
Comment (ItzLevvie) [inline manifests/a/ArtisteerLimited/Nicepage/4.0.3.4.0.3/ArtisteerLimited.Nicepage.locale.en-US.yaml:5]:
```suggestion
PackageVersion: 4.0.3
```

File: manifests/a/ArtisteerLimited/Nicepage/4.0.3.4.0.3/ArtisteerLimited.Nicepage.yaml [hunk 2]
```diff
@@ -0,0 +1,9 @@
+# Created using wingetcreate 0.4.3.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.0.0.schema.json
+
+PackageIdentifier: ArtisteerLimited.Nicepage
+PackageVersion: 4.0.3.4.0.3
```
Comment (ItzLevvie) [inline manifests/a/ArtisteerLimited/Nicepage/4.0.3.4.0.3/ArtisteerLimited.Nicepage.yaml:5]:
```suggestion
PackageVersion: 4.0.3
```

File: manifests/a/ArtisteerLimited/Nicepage/4.0.3.4.0.3/ArtisteerLimited.Nicepage.installer.yaml [hunk 3]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 0.4.3.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+
+PackageIdentifier: ArtisteerLimited.Nicepage
+PackageVersion: 4.0.3.4.0.3
```
Comment (ItzLevvie) [inline manifests/a/ArtisteerLimited/Nicepage/4.0.3.4.0.3/ArtisteerLimited.Nicepage.installer.yaml:5]:
```suggestion
PackageVersion: 4.0.3
```

**Review by ItzLevvie — CHANGES_REQUESTED** [review]
This must match Control Panel.

**Comment by Xanthus58** [comment]
Sorry for the long wait had to do some stuff irl and get my laptop charged

**Review by ItzLevvie — APPROVED** [review]

**Pull request merged**